docs/application_profile.md

```diff
@@ -54,14 +54,15 @@ The DLME Prototype Application Profile a subset of the [Europeana Data Model](ht
 
 | Solr field name  | Property          | Expectation | Cardinality | Value type  | Definition | Notes                      |
 | ---------------- | ----------------- | ----------- | ----------- | ----------- | ---------- | -------------------------- |
+| cho_aat_material | `http://vocab.getty.edu/page/aat/300010357` | R | 0...n | literal | Getty Art & Architecture material facet. | |
```

Comment:
The property column should match to a specific RDF property; I can take a look and make a recommendation later.

Reply:
is it `aat:300010357`?

Reply:
No, that's for "Materials" as a specific facet within the AAT hierarchy. Is this intended to be a controlled value? It feels similar to `dct:medium` as used below (`cho_medium`) although I understand that we wouldn't necessarily want to duplicate use of that.

Reply:
yes it is a controlled value; it is our effort to normalize cho_medium. The values are from the AAT Materials facet.